src/hooks/useFetchDustInfo.ts

```diff
@@ -24,30 +24,31 @@ export const cityGroup = [
   { cityName: '세종', cityNumber: 16 },
 ];
 
-const useFetch = () => {
-  const [data, setData] = useState<SidoDust[] | []>([]);
-
-  const fetchData = async () => {
-    const result = await Promise.all(
-      cityGroup.map((v) =>
-        axios
-          .get(
-            `${VITE_OPEN_URL}?sidoName=${v.cityName}&pageNo=1&numOfRows=100&returnType=json&serviceKey=${VITE_API_KEY}&ver=1.0`
-          )
-          .then((res) => {
-            return res.data.response.body;
-          })
-      )
-    );
-
-    setData(result);
+const useFetchDustInfo = () => {
+  const [dustInfo, setDustInfo] = useState<SidoDust[] | []>([]);
+
+  const fetchDustInfo = async () => {
+    try {
+      const result = Promise.all(
+        cityGroup.map((v) =>
+          axios
+            .get(
+              `${VITE_OPEN_URL}?sidoName=${v.cityName}&pageNo=1&numOfRows=100&returnType=json&serviceKey=${VITE_API_KEY}&ver=1.0`
+            )
+            .then((res) => {
+              return res.data.response.body;
+            })
+        )
+      );
+      setDustInfo(await result);
+      return result;
+    } catch (err) {
+      console.log(err);
```

Comment:
```suggestion
      console.error(err);
```

console에서 log보다 error을 쓰면 더 좋을 것 같아요. 아니면 console을 굳이 안 넣어도 되기도하구요!